mongo/service.go

```diff
@@ -183,7 +187,10 @@ func newConf(args ConfigArgs) common.Conf {
 			" --smallfiles"
 	} else {
 		mongoCmd = mongoCmd +
-			" --storageEngine wiredTiger"
+			" --storageEngine wiredTiger" +
```

Comment:
It was supposed to, but for some reason its not, ill fix that since its the intended behavior